e2e/utils.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2017 The Kubernetes Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"flag"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var url string
+var tillerHost string
+var helmBinPath string
+var localTiller bool
+
+func init() {
+	flag.StringVar(&url, "cluster-url", "http://127.0.0.1:8080", "apiserver address to use with restclient")
+	flag.StringVar(&tillerHost, "tiller-host", "", "tiller address")
+	flag.StringVar(&helmBinPath, "helm-bin", "helm", "helm binary to test")
+	flag.BoolVar(&localTiller, "local-tiller", false, "wait for tiller pod")
+}
+
+func LoadConfig() *rest.Config {
+	config, err := clientcmd.BuildConfigFromFlags(url, "")
+	Expect(err).NotTo(HaveOccurred())
+	return config
+}
+
+func KubeClient() (*kubernetes.Clientset, error) {
+	config := LoadConfig()
+	clientset, err := kubernetes.NewForConfig(config)
+	Expect(err).NotTo(HaveOccurred())
+	return clientset, nil
+}
+
+func DeleteNS(clientset kubernetes.Interface, namespace *v1.Namespace) {
+	defer GinkgoRecover()
+	pods, err := clientset.Core().Pods(namespace.Name).List(metav1.ListOptions{})
+	Expect(err).NotTo(HaveOccurred())
+	for _, pod := range pods.Items {
```

Comment:
You shouldn't have to do this if I remember correctly. Deleting the namespace should just delete the pods

Reply:
That's true AFAIK. Will fix.